blacklist-call/functions/blacklist-call.js

```diff
@@ -0,0 +1,20 @@
+exports.handler = function(context, event, callback) {
+    let blacklist = event.blacklist || context.BLACKLIST || '';
+    const blacklistArray = blacklist.toString().split(',').map(num => num.trim());
+
+    let twiml = new Twilio.twiml.VoiceResponse();
+    let blocked = true;
+    if (blacklistArray.length) {
+      if (!blacklistArray.includes(event.From)) {
+        blocked = false;
+      }
+    }
```

Comment:
If the blacklistArray is empty (ie nothing is blacklisted) this will cause all calls to be blocked. I would factor the conditional the other way around, which would read better too. You can also do the conditional in one go.

```javascript
let blocked = false;
if(blacklistArray.length > 0 && blacklistArray.includes(event.From)) {
  blocked = true;
}
```

Reply:
updated with empty blacklist test